userguide/services.rst

```diff
@@ -1603,6 +1610,13 @@ unless there is a specific need.**
   overhead.
 
 
+.. needs link to smb1 knowledgebase article, when it is published.
+
+Even though the protocol is deprecated and vulnerable, some clients can
```

Comment:
Suggestion: `` The SMB1 protocol is deprecated and vulnerable. Some older clients still require it.  Before enabling SMB1, see `Why You Should Not Use SMB1 <>`__ for further information.